pyanaconda/modules/payload/payloads/live_os/live_os.py

```diff
@@ -95,31 +95,28 @@ def _check_source_availability(self, message):
         if not self._image_source:
             raise SourceSetupError(message)
 
-    @property
-    def space_required(self):
-        """Get space required for the source image.
-
-        TODO: Add missing check if source is ready. Until then you shouldn't call this when
-        source is not ready.
-
-        TODO: This is not that fast as I thought (a few seconds). Caching or task?
-
-        :return: required size in bytes
-        :rtype: int
-        """
+    @staticmethod
+    def _get_requiered_space():
```

Comment:
a typo in the name, should  be _get_required_space ?

Reply:
Fixed.